apps/prairielearn/src/pages/instructorFileEditor/instructorFileEditor.ts

```diff
@@ -22,26 +20,60 @@ import { FileModifyEditor } from '../../lib/editors.js';
 import { deleteFile, getFile, uploadFile } from '../../lib/file-store.js';
 import { idsEqual } from '../../lib/id.js';
 import { getPaths } from '../../lib/instructorFiles.js';
-import { getJobSequenceWithFormattedOutput } from '../../lib/server-jobs.js';
+import {
+  getJobSequenceWithFormattedOutput,
+  type JobSequenceWithFormattedOutput,
+} from '../../lib/server-jobs.js';
+
+import { InstructorFileEditor } from './instructorFileEditor.html.js';
 
 const router = express.Router();
 const sql = sqldb.loadSqlEquiv(import.meta.url);
 const debug = debugfn('prairielearn:instructorFileEditor');
 
+interface FileEdit {
```

Comment:
This is going to be reviewed in a future PR that is going to hopefully simplify some of this data, synchronize types with HTML.TS, and incorporate the changes from #10224 into this code.